core/dictgen/src/rootcling_impl.cxx

```diff
@@ -4876,7 +4933,8 @@ int RootClingMain(int argc,
          if (modGen.IsPCH()) {
             if (!GenerateAllDict(modGen, CI, currentDirectory)) return 1;
          } else if (getenv("ROOT_MODULES")) {
-            if (!GenerateModule(modGen, CI)) return 1;
+            if (!GenerateModule(modGen, resourceDir, interp, linkdefFilename, moduleName.str()))
+               return 1;
```

Comment:
If we return only 0 or 1 shouldn't we prefer true/false?

Reply:
This return is in the >1000 line "main" method that returns something like an exit code and `GenerateModule` already returns true/false :). But it's on my list of things I would like to change in rootcling.